src/users-plugin.js

```diff
@@ -73,7 +78,7 @@ module.exports = {
     server.route({
       method:'DELETE',
       path:'/{userId}',
-      handler(request, h) {
+      handler(request, responseHandler) {
```

Comment:
No good reason. I was originally expecting to use `responseHandler` extensively as I got accustomed to `hapi` patterns. But that didn't pan out and I should have removed all references.

I'm also realizing that had I added a linter, it would have picked up on `responseHandler` as an unused var as well.